src/gui/settingswindow.h

```diff
@@ -0,0 +1,87 @@
+/*
+ * @file settingswindow.h
+ * @brief Header file for the settings GUI page
+ * @copyright Copyright (C) 2021  Andrew Ritchie
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef SRC_GUI_SETTINGSWINDOW_H_
+#define SRC_GUI_SETTINGSWINDOW_H_
+
+namespace Ui {
+class SettingsWindow;
+}
+
+/*
```

Comment:
You need to make sure all comments for Doxygen start with `/**` so that it picks it up correctly. You can follow the steps I added [here](https://ese-peasy.github.io/PosturePerfection/guidance.html#locally-generating-doxygen-documentation) to generate the Doxygen documentation locally and ensure everything looks correct (make sure you don't push anything in the `html/` directory after doing this)